docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,286 @@
+
+**Set the Correct Amount To Send:**
+
+By default the scripts send the wallet's entire AVAX balance:
+
+```js
+const balance: BN = new BN(getBalanceResponse.balance);
+const amount: BN = balance.sub(fee);
+```
+
+To send a different amount, please replace the code above with the following. Below sets a new value of 0.01 AVAX (`10000000` GWEI). Value is set in GWEI format where `1,000,000,000` GWEI = 1 AVAX
+
+```js
+const value: BN = new BN("10000000");
+const amount: BN = value.sub(fee);
+```
+
+:::tip
+Snowtrace provides a [unit converter](https://snowtrace.io/unitconverter) between different units
+:::
+
+Run the export script:
+
+```sh
+$ ts-node examples/avm/buildExportTx-cchain-avax.ts
+```
+
+This returns:
+
+```sh
+Success! TXID: LjWWen7AwA3gP2op2V2Gwyoh9iNbPp9ZggFqAAs6YtPUscWvg
+```
+
+#### Verify the Transaction
+
+You can now pass this txID `LjWWen7AwA3gP2op2V2Gwyoh9iNbPp9ZggFqAAs6YtPUscWvg` into [examples/avm/getTx.ts](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/getTx.ts), plus other similar network settings, then you can run
+
+```zsh
+$ ts-node examples/avm/getTx.ts
+```
+
+which returns:
+
+```js
+{
+  unsignedTx: {
+    networkID: 5,
+    blockchainID: '2JVSBoinj9C2J33VntvzYtVJNZdN2NKiwwKjcumHUWEb5DbBrm',
+    outputs: [],
+    inputs: [ [Object] ],
+    memo: '0x41564d207574696c697479206d6574686f64206275696c644578706f7274547820746f206578706f7274204156415820746f2074686520432d436861696e2066726f6d2074686520582d436861696e',
+    destinationChain: 'yH8D7ThNJkxmtkuv2jgBa4P1Rn3Qpr4pPr7QYNfcdoS6k6HWp',
+    exportedOutputs: [ [Object] ]
+  },
+  credentials: [
+    {
+      fxID: 'spdxUxVJQbX85MGxMHbKw1sHxMnSqJ3QBzDyDYEP3h6TLuxqQ',
+      credential: [Object]
+    }
+  ]
+}
+```
+
+#### Import the Avax Token From X-Chain to C-Chain
+
+Select the [**`examples/evm`**](https://github.com/ava-labs/avalanchejs/tree/master/examples/evm) folder to view the AvalancheJS C-Chain examples. To import AVAX to the C-Chain from the X-Chain, select [`evm/buildImportTx-xchain.ts`](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildImportTx-xchain.ts)
+
+Copy the [network setting from above](#network-setting) into `evm/buildImportTx-xchain.ts`.
+
+Navigate to this part of the code and ensure that the `cHexAddress`(_Your C-Chain wallet address_) and `private key` are correct:
+
+```ts
+const cHexAddress: string = "<YOUR-CCHAIN-WALLET-ADDRESS-HERE>";
+const privKey: string = "<YOUR-PRIVATE-KEY-HERE>";
+```
+
+Run the import script:
+
+```sh
+$ ts-node examples/evm/buildImportTx-xchain.ts
+```
+
+This returns:
+
+```sh
+Success! TXID: ft33QHCH542wWUwrjdVf9APtAjPWqzqbqE43nCRECdbzhzVub
+```
+
+That's it! You've transferred AVAX from the X-Chain to C-Chain!
+
+You can verify this TX by copy / pasting the import TXID into [Avascan](https://testnet.avascan.info/blockchain/c/tx/ft33QHCH542wWUwrjdVf9APtAjPWqzqbqE43nCRECdbzhzVub)
```

Comment:
This example shows of importing 9.999 [AVAX](https://testnet.avascan.info/blockchain/x/asset/U8iRqJoiJm8xZHAacmvYyZVwqQx6uDNtQeP3CQ6fcgQk3JqnK) while this examples uses value of 0.01 avax for transfer. Please make them consistent.